docs/GettingStartedDocs/IntelSDKPortingGuide.md

```diff
@@ -0,0 +1,214 @@
+# Intel SGX SDK to OpenEnclave SDK Porting Guide
```

Comment:
There are two separate Intel SGX SDKs... "SGX SDK for Linux" and "Intel SGX SDK for Windows".  We should avoid the unqualified term "Intel SGX SDK" throughout as it is a source of confusion.

I would like porting guide(s) for both.  Is the intent to put both into this document or have two documents?
if the latter, please change the title of the doc. 

And independent of the doc question, is this PR intending to cover both Windows and Linux?  If not, please change the title of the PR.

Reply:
The ultimate goal will be porting guides for both platforms but this PR and this doc is focus on Linux. I'll update the titles of both doc and PR.

Reply:
Great, thanks.  We should file a separate github issue to track the porting guide for Windows then.